config/env/development.example.env

```diff
@@ -33,3 +33,6 @@ RAISE_DELIVERY_ERRORS="false"
 PSQL_HOST=""
 PSQL_USERNAME=""
 PSQL_PASSWORD=""
+
+# Microsoft Sentiment Analysis
+OCP_APM_SUBSCRIPTION_KEY=""
```

Comment:
Do we still need this? We're not gonna test out Sentiment Analysis in this PR, right?